src/main/java/AddCommand.java

```diff
@@ -0,0 +1,88 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+/**
+ * Class for the command that adds a task into the tasklist.
+ */
+public class AddCommand extends Command {
+
+    private String input;
+
+    /**
+     * Constructor for the AddCommand object.
+     *
+     * @param ipt the string representation of the command input.
+     */
+    public AddCommand(String ipt) {
+        input = ipt;
```

Comment:
Not compulsory, but would be good to have a 'this' keyword to reference the current object. 
e.g. this.input = ipt
Would become ambiguous if there are similar variable names.